examples/torchvision_dataset.ipynb

```diff
@@ -0,0 +1,184 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "8fa4a083-1a56-419f-9e3f-fa93acd45405",
+   "metadata": {},
+   "source": [
+    "# Run CleanVision on Torchvision dataset"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6c52419e-f8d7-4f73-8a5d-c6d5cf90fa77",
+   "metadata": {},
+   "source": [
+    "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/cleanlab/cleanvision/blob/main/examples/torchvision_dataset.ipynb) "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2c0632a9-0820-4143-aaba-3e64e8a033f2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install -U pip\n",
+    "!pip install cleanvision[pytorch]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "dfcd1327-924c-47ae-834d-3b9d922800f4",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "**After you install these packages, you may need to restart your notebook runtime before running the rest of this notebook.**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "50a8cd5a-899e-4a58-84f7-fb1ddf4bfcf2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from torchvision.datasets import CIFAR10\n",
+    "from torch.utils.data import ConcatDataset\n",
+    "from cleanvision.imagelab import Imagelab"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7dcf23f9-c031-4bdf-b63c-5eb02003321b",
+   "metadata": {},
+   "source": [
+    "### Download dataset and concatenate all splits"
+   ]
```

Comment:
somewhere notebook should clarify: This is a classification dataset, but CleanVision can be used to audit images from any type of dataset (including supervised or unsupervised learning).